src/cli/env.js

```diff
@@ -1,5 +1,13 @@
 const parseEnv = () => {
-    // Write your code here 
+    let resStr = '';
+    const env = process.env;
+
+    Object.keys(env).forEach(key => {
+        if(key.includes("RSS"))
```

Comment:
It would be less error prone if you use .startsWith('RSS_') inside condition.
For example, if I have env variables such as "some_env_RSS_somevalue" or "SomeEnvRSSsomeSalue" you code will include it in the output. This is incorrect behaviour. I will not deduct any point, but share with you how you can improve it.